site/docs/v1/tech/shared/_premium-edition-blurb.adoc

```diff
@@ -1 +1,4 @@
-*This is a paid edition feature and is unavailable in the community edition. Please visit link:/pricing/editions/[our pricing page] to learn more about paid editions.*
+[NOTE]
```

Comment:
FYI that this is going to change everywhere that this blurb is used. I like the note, but we should look at all the pages and see if we need to move things around a bit. For example on http://localhost:4000/docs/v1/tech/account-management/authenticator-two-factor/ we get two bubbles close to each other. Would prefer they are separated by some text. 

Can you put the blurb just on top of the `Enable MFA Method on Tenant` header?

![Screen Shot 2021-05-12 at 3 41 05 PM](https://user-images.githubusercontent.com/91825/118047756-78bb1880-b338-11eb-9f3a-0d066f645155.png)

There are also other places (grep is your friend) you'll need to check. I know that there are some places where we **bold** the fact that TOTP isn't a paid edition feature, so you'll want to deal with that too. See https://fusionauth.io/docs/v1/tech/apis/two-factor/ for instance.

Reply:
> I know that there are some places where we bold the fact that TOTP isn't a paid edition feature, so you'll want to deal with that too. 

#722 Linked PR - not sure how best to address.  Let me know if you had something specific in mind.
I suppose one option would be to write a separate `site/docs/v1/tech/shared/_premium-edition-blurb.adoc` => `site/docs/v1/tech/shared/_premium-edition-blurb-with-free-adjacent.adoc`

### Current look
![image](https://user-images.githubusercontent.com/16090626/119763487-dd17c500-be6c-11eb-9681-4bce03a6b708.png)

### A possible suggestion
I think that the only two places that will need this are 
- https://site-local.fusionauth.io/docs/v1/tech/tutorials/two-factor/
- https://fusionauth.io/docs/v1/tech/apis/two-factor/ 

But what are your thoughts on this additional callout?  Can talk live if that helps 🙂 

![image](https://user-images.githubusercontent.com/16090626/119764722-26691400-be6f-11eb-85a8-2fbece90b311.png)

Reply:
The more that I think about this, I think that it might be ok like it is above ☝🏻  b/c it's only two places and just one line of bolded text